src/JSInjection/Modal/ToolbarButtons.js

```diff
@@ -1,30 +1,91 @@
 import * as s from "../../zeeguu-react/src/reader/ArticleReader.sc"
-import strings from "../../zeeguu-react/src/i18n/definitions"
 import {toggle} from "../../zeeguu-react/src/reader/ArticleReader"
+import * as React from 'react';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+import { styled } from '@mui/material/styles';
+import FormHelperText from '@mui/material/FormHelperText';
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const t = createTheme({
+  components: {
+    MuiSwitch: {
+      styleOverrides: {
+        switchBase: {
+          // Controls default (unchecked) color for the thumb
+          color: "#ccc"
+        },
+        colorPrimary: {
+          "&.Mui-checked": {
+            // Controls checked color for the thumb
+            color: "#ffbb54"
+          }
+        },
+        track: {
+          // Controls default (unchecked) color for the track
+          opacity: 0.7,
+          backgroundColor: "#999999",
+          ".Mui-checked.Mui-checked + &": {
+            // Controls checked color for the track
+            opacity: 0.5,
+            backgroundColor: "#ffbb54"
+          }
+        }
+      }
+    }
+  }
+});
+
+const Android12Switch = styled(Switch)(({ theme }) => ({
```

Comment:
why do we have something that has Android in the name?

Reply:
It's a component from Mui